package.json

```diff
@@ -146,5 +146,8 @@
     "download-build": "node ./scripts/release/download-experimental-build.js",
     "download-build-for-head": "node ./scripts/release/download-experimental-build.js --commit=$(git rev-parse HEAD)",
     "download-build-in-codesandbox-ci": "cd scripts/release && yarn install && cd ../../ && yarn download-build-for-head || yarn build-combined --type=node react/index react-dom scheduler"
+  },
+  "resolutions": {
+    "**/@playwright/test/**/react-is": "^16.0.0"
```

Comment:
Could this resolution be declared in the `react-devtools-inline` package (since that's the only thing that uses @playwright)? Just curious.

Reply:
Good question. I always declare them in the root. I tested it quickly and it didn't have any effect when declared in a nested package.

Though it may be preferred to declare it at the root in case another package uses `@playwright/test`?

Reply:
Yeah that's fair.